build-system/test-configs/config.js

```diff
@@ -252,7 +263,6 @@ const htmlFixtureGlobs = [
   '!examples/amp-redbull-player.amp.html',
   '!examples/amp-script/example.amp.html',
   '!examples/amp-script/example.sandboxed.amp.html',
-  '!examples/amp-script/hello-world.html',
```

Comment:
Empty except for an empty JS reference so it doesn't do anything, not an AMP doc, not referenced by anything.